sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoTest.java

```diff
@@ -1306,11 +1307,15 @@ public void testParDoShiftTimestamp() {
   @Category(NeedsRunner.class)
```

Comment:
Mostly they're meant to be an interestingness barrier, excepting things like `TestStream`. I'll happily change these, as there doesn't appear to be any equivalent coverage.